### ISSUE TITLE
(UWP) homebrew games wont delete or uninstall

### ISSUE THREAD

**Issue by dominater01** [hrydgard/ppsspp#11904]
i dont know if this is just an xbox one issue or not but when you have homebrew and you press y on it and goto delete it does nothing and still shows up and when you download homebrew a uninstall option shows up on the store and that does nothing

build version 1.8.0-3

this has always been a problem for the uwp builds

**Comment by leopard20** [comment]
I also can't delete dump files I load up manually (esp. when they show up in Recent list). Perhaps because their files are not the located in PPSSPP>GAME?

**Comment by hrydgard** [comment]
Yeah, both these issues (which are probably entirely separate, @leopard20  you mean not only on UWP, right?) need looking into.

**Comment by unknownbrackets** [comment]
It's /very/ possible I might've forgotten to add delete handling to some switch for the GE DUMP filetype...

-[Unknown]

**Comment by leopard20** [comment]
@hrydgard Yes. Forgot to mention. I meant on Android!

**Comment by hrydgard** [comment]
Any remaining problems are moved to 1.10.

**Comment by dominater01** [comment]
this works on xbox one now both uninstall and delete except uninstall for cso/iso files delete is fine

**Comment by unknownbrackets** [comment]
So the title is fixed now, it's just that deleting iso/cso files still doesn't work - right?

-[Unknown]

**Comment by dominater01** [comment]
deleting iso/cso is fine but on the homebrew store it wont uninstall

**Comment by NABN00B** [comment]
Does this still happen in 1.18.1?
